heat_infoblox/object_manipulator.py

```diff
@@ -36,32 +36,68 @@ def get_member(self, member_name, return_fields=None, extattrs=None):
             'member', obj, return_fields, extattrs
         )
 
-    def create_member(self, name=None, platform='VNIOS',
-                      mgmt=None, lan1={}, lan2=None,
-                      nat_ip=None):
+    def create_member(self, name=None, platform='VNIOS', ha_pair=False,
+                      vip=None, mgmt=None, lan2=None, nat_ip=None,
+                      use_v4_vrrp=False,
+                      node1_ha=None, node2_ha=None,
+                      node1_lan1=None, node2_lan1=None,
+                      node2_mgmt=None, vrid=None, lan2_vrid=None):
         member_data = {'host_name': name, 'platform': platform}
         extra_data = {}
 
-        if lan1.get('ipv4', None):
-            extra_data['vip_setting'] = lan1['ipv4']
-        if lan1.get('ipv6', None):
-            extra_data['ipv6_setting'] = lan1['ipv6']
+        if ha_pair:
+            node1 = {}
+            if node1_ha and node1_ha.get('ipv4', None):
+                node1['ha_ip_address'] = node1_ha['ipv4']['address']
```

Comment:
This check 'node1_ha.get('ipv4', None)' will be false in case of 'node1_ha['ipv4'] is None', so line 51 will be executed and it will not fail here.
However, if ip address for ha pair is not provided it will probably fail on member creation.